teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Kindly use the colon notation to assign the `parent_permission` parameter for the super to avoid ArgumentError as you already use the colon notation in the parent class(Person), so it will be like:
```ruby
super(age, name, parent_permission: parent_permission)
```

Reply:
Thank you for that tip.